### PR TITLE
[#188] Supports usage camelized `inheritdoc` annotation

### PR DESCRIPTION
**Pull request by andrew-demb**
Use case-insensitive function to check - `stripos`

### CODE REVIEW

File: tests/Sniffs/TypeHints/data/typeHintDeclarationNoErrors.php [hunk 1]
```diff
@@ -631,4 +631,19 @@ public function inheritdocAnnotation($a)
 		return true;
 	}
 
+    /**
+     * @inheritDoc
+     */
+	public function camelizedInheritdoc($a)
+    {
```
Comment (kukulich) [inline tests/Sniffs/TypeHints/data/typeHintDeclarationNoErrors.php]: It looks as spaces here. We use tabs for indenting.

**Comment by andrew-demb** [comment]
Link: #188

> Offtop: I don't know why but in repo code pushed with wrong formatting. My IDE show all as OK.

**Comment by kukulich** [comment]
> I don't know why but in repo code pushed with wrong formatting. My IDE show all as OK.

@andrew-demb I looks you use spaces instead of tabs.

File: tests/Sniffs/TypeHints/data/typeHintDeclarationNoErrors.php (hunk 1 above)
Comment (andrew-demb) [reply tests/Sniffs/TypeHints/data/typeHintDeclarationNoErrors.php]: Done.

File: tests/Sniffs/TypeHints/data/typeHintDeclarationNoErrors.php [hunk 2]
```diff
@@ -631,4 +631,19 @@ public function inheritdocAnnotation($a)
 		return true;
 	}
 
+	/**
+	 * @inheritDoc
```
Comment (kukulich) [inline tests/Sniffs/TypeHints/data/typeHintDeclarationNoErrors.php]: @andrew-demb I'm sorry, i missed this in first review. There should be `{@inheritDoc}`

Comment (andrew-demb) [reply tests/Sniffs/TypeHints/data/typeHintDeclarationNoErrors.php]: Done.

**Pull request merged**